django/api/services/spreadsheet_uploader_prep.py

```diff
@@ -245,7 +245,9 @@ def validate_phone_numbers(df, *columns, **kwargs):
         series = df[column]
         for index, phone_number in series.items():
             formatted_number = str(phone_number).strip().replace('-', '')
-            if formatted_number == '' or len(formatted_number) != 10 or int(formatted_number[:3]) not in AREA_CODES:
+            if len(formatted_number) != 10 or int(formatted_number[:3]) not in AREA_CODES:
```

Comment:
Looking at the card, I think they want empty values to not register as errors, so I think we have to take the negation of ` pd.notna(formatted_number) and formatted_number != ''` instead:
```
for index, phone_number in series.items():
        formatted_number = str(phone_number).strip().replace('-', '')
        if pd.isna(formatted_number) or formatted_number == '':
            continue # skip the rest of the loop and go to the next iteration
        if len(formatted_number) != 10 or int(formatted_number[:3]) not in AREA_CODES:
            indices.append(index + kwargs.get("indices_offset", 0))
```

Reply:
Yup, accidentally did it a little backwards. Should be fixed now.